20251122 en/zoom/create room/registration

Makoto-SHIMAOKA opened the pull request.

Makoto-SHIMAOKA committed:
create index.mdx

Makoto-SHIMAOKA committed:
upload img

Makoto-SHIMAOKA committed:
update index.mdx

Makoto-SHIMAOKA committed:
update index.mdx

Makoto-SHIMAOKA committed:
update manual approval

Makoto-SHIMAOKA committed:
update index.mdx

yoshino-yumi commented on src/pages/en/zoom/create_room/registration/ChangeToRegister.png:
meeting情報はグレーで隠した方がいいかもしれません

Yuki0631 replied:
修正しました。

yoshino-yumi commented on src/pages/en/zoom/create_room/registration/index.mdx:
```diff
@@ -0,0 +1,312 @@
+1. Once you click on the "Required" checkbox, three options will appear. Please select one.
+   -  **Attendees register once and can attend any of the occurrences – once a prospective participant registers, they can attend all sessions.
+   -  **Attendees need to register for each occurrence to attend – prospective participants need to perform the registration procedure each time.
+   -  **Attendees register once and can choose one or more occurrences to attend – prospective participants can select which session they want to register for.
```
```suggestion
   -  **Attendees register once and can choose one or more occurrences to attend** – prospective participants can select which session they want to register for.
```
これで太字になるかと

yoshino-yumi commented on src/pages/en/zoom/create_room/registration/ResendMailChooseMeeting.png:
meeting idは隠したほうがいいかもしれません

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Makoto-SHIMAOKA committed:
Update src/pages/en/zoom/create_room/registration/index.mdx

Co-authored-by: Yumi Yoshino <yoshinoyumi@g.ecc.u-tokyo.ac.jp>

Yuki0631 committed:
revise the first two images

Yuki0631 committed:
revise english images

Yuki0631 committed:
change mdx to md

Yuki0631 commented on src/pages/en/zoom/create_room/registration/FormQuestionnaireTab.png:
修正しました。

Yuki0631 committed:
revise english images

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+The meeting host creates a registration link using this feature.
+Subsequently, the host sends the relevant link to prospective participants and asks them to complete the registration procedure.
+Prospective participants who wish to attend the meeting are required to access the registration page provided by the host and complete the registration procedure.
+Prospective participants who register are automatically approved to join the meeting on completing the procedure and receive a confirmation email containing an individualised link.
```
```suggestion
Prospective participants who register are automatically approved to join the meeting upon completing the procedure and receive a confirmation email containing an individualised link.
```

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+1. Prospective participants can perform the necessary registration procedures through the "Registration Link".
+   Participants who have completed the registration procedure will automatically receive a confirmation email containing the individualised link.
+
+※For the host to be able to individually decide whether or not a prospective participant will receive the confirmation email or not, please use the "Manual Approval" option mentioned below.
```
```suggestion
※To individually decide whether a prospective participant will receive a confirmation email or not, please use the "Manual Approval" option mentioned below.
```

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+   ![](RegisterLink.png){:.medium}
+
+※If the email addresses of prospective participants are not in the format of `0123456789@utac.u-tokyo.ac.jp`, you may still be able to register in the Zoom system (if you have the Zoom account with `u-tokyo.ac.jp` at the end). 
+However, because using the UTokyo Account (10-digit Common ID+`@utac.u-tokyo.ac.jp`) as the Zoom account is standard at the University of Tokyo, please enter in the format of `0123456789@utac.u-tokyo.ac.jp`.
```
```suggestion
However, because using the UTokyo Account (10-digit Common ID+`@utac.u-tokyo.ac.jp`) as the Zoom account is standard at the University of Tokyo, please enter it in the format of `0123456789@utac.u-tokyo.ac.jp`.
```

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+
+Using Zoom's participant registration feature allows you to accomplish the following.
+
+-  Using the "Manually Approve" option to prevent registered prospective participants from receiving the join link until approved by the host.
```
```suggestion
-  Use the "Manually Approve" option to prevent registered prospective participants from receiving the join link until approved by the host.
```

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+1. Click on the "Edit" button on the right of "Registration Options".
+   ![](ManualEditOption.png){:.medium}
+1. On the displayed screen, please select "Manually Approve". 
+   ![](ManualApplied.png){:.medium}
```
```suggestion
```

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+1. On the displayed screen, please select "Manually Approve". 
+   ![](ManualApplied.png){:.medium}
+1. Click on the "Save All" button.
+   ![](ManualAccepted.png){:.medium}
```
```suggestion
```

Koyuki-Komachi commented on src/pages/en/zoom/create_room/registration/index.md:
```diff
@@ -0,0 +1,314 @@
+   ![](ManualApplied.png){:.medium}
+1. Click on the checkbox on the left of the prospective participants you want to reject and click on the "Deny" button.
+   ![](ManualRejected.png){:.medium}
+1. The screen for "Send Registration Refusal Email" will be displayed. Enter the message you want to convey to the rejected prospective participant, and click on "Send". The rejected prospective participant will receive the registration rejection email that contains what you enter here.
```
```suggestion
1. The screen for "Send Registration Denial Email" will be displayed. Enter the message you want to convey to the rejected prospective participant, and click on "Send". The rejected prospective participant will receive the registration rejection email that contains what you enter here.
```

Koyuki-Komachi committed:
update screenshots to latest version

replace FormQuestionareOriginalQuestionSave.png

replace FormRegisterTab.png

replace FormRegisterTab.png

replace FormRegisterTab.png

replace RegisterTab images

replce ViewReportStatus.png and ViewReportMeeting.png

Koyuki-Komachi committed:
modify "How to Retrieve a List of Answers" section

Koyuki-Komachi committed:
update index.md
